README.md

```diff
@@ -60,36 +60,36 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Awesome Books Project] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
```

Comment:
- [ ] @Lasten-Ernest kindly remove all the instructions statements, this will make your readme look more professional and neat.